### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by hagamingpc** [sumatrapdfreader/sumatrapdf#4488] [General]
Version v3.6.16616 64-bit, pre-release. 
Dear SumatraPDF Team,

I've been using SumatraPDF for a while now and I really like its simplicity and efficiency. However, I've noticed that the application takes a long time to load certain PDF files and sometimes becomes unresponsive, requiring me to force close it. I'm using the Windows 11 Insider Preview, so I understand it might be related to the operating system's instability or compatibility. Just wanted to bring this to your attention in case it's something that could be addressed in future updates. I'd appreciate any advice or fixes to improve performance, especially on preview versions of Windows.

Thank you for your continued work on SumatraPDF—it's an excellent tool, and I look forward to future improvements. 
Best regards,

**Comment by GitHubRulesOK** [comment]
Hmm we are on 16766 and speed is variable during pre-release changes so you should be testing the one that is 150 versions newer. 

However I suspect your "subjective analysis" (gut feelings) may not be unfounded as with more and more added features there is often a cost in terms of response. 

If there is a definite **repeatable** lag in one type of PDF when compared with a similar one (always hard to determine as there are so many variations) then consider if a sample may help to check why.

**Reply by hagamingpc** [reply]
I'm experiencing the issue in every time, in every file, where my system always loads slowly and shows a "Not Responding" message for a few minutes before it finally opens files. This happens regularly and is becoming frustrating. minimum times require to open a file is 2 minutes.

**Comment by kjk** [comment]
If you don't provide the document that has this problems, there's nothing I can do.

Also, if your files are on a network drive (including OneDrive), they might load slowly if the network is slow.

**Reply by hagamingpc** [reply]
I'm experiencing the issue in every time, in every file, where my system always loads slowly and shows a "Not Responding" message for a few minutes before it finally opens files. This happens regularly and is becoming frustrating. minimum times require to open a file is 2 minutes.

**Reply by kjk** [reply]
Use `Ctrl + K`, `show logs` and post the relevant part that says sth. like:
```
LoadDocument: 11.59 ms, 35 pages for 'C:\Users\kjk\Downloads\vdoc.pub_fly-high-level-3-fun-grammar-teachers-guide.pdf'
```

**Reply by hagamingpc** [reply]
LoadDocument: 19.45 ms, 2 pages for 'C:\Users\HAGam\Downloads\Class 10 Central Assessment 2024 mghs.pdf'

**Reply by kjk** [reply]
Per your log:
```
LoadDocument: 9.83 ms, 1 pages for 'C:\Users\HAGam\Downloads\students.pdf'
```

Sumatra took 9.82 milliseconds to load the file.

Whatever is going on, it's not caused by SumatraPDF loading the file slowly.

It could be bad anti-virus software, actual virus or some other misconfiguration but it's clearly not SumatraPDF loading file slowly.

**Reply by hagamingpc** [reply]
LoadDocument: 147.55 ms, 302 pages for 'D:\Download\Documents\G_Eng_Classx.pdf'

**Reply by hagamingpc** [reply]
okay

**Comment by hagamingpc** [comment]
thanks for the info. but i don't know what is causing this problem

**Reply by GitHubRulesOK** [reply]
What exactly is D:\   what type of device is it onboard / USB chip / USB drive  / NAS / other desktop  ?
My V:\ drive is another desktop and very quick to load from there 2 seconds for 250 MB PDF download from one room to another.

However in your case it seems like there is a 3rd party product interfering. What is running that may be controlling file access ?

**Reply by GitHubRulesOK** [reply]
@hagamingpc 
Just out of interest to ensure nothing in sumatrapDF is having any effect.
Backup your settings file and delete the working copy (thus it has to write a new one on first restart) open any file and see if it is as slow as you currently notice for such file.

perhaps review using task manager what processes kick in to action when opening a file with SumatraPDF
When I open that remote file both explorer and antimalware consume more CPU time than SumatraPDF does for the 2 seconds.

**Reply by hagamingpc** [reply]
D:\ onboard (system partition
![Screenshot 2024-08-27 212301](https://github.com/user-attachments/assets/41466425-58a3-4bbc-ae31-af75aa984fce)
)

**Comment by hagamingpc** [comment]
LoadDocument: 34.53 ms, 2 pages for 'C:\Users\HAGam\Downloads\Attendance Sheet.1.pdf'
as per log, this pdf file took 34 ms but it takes 1minutes 10 sec to open

**Reply by GitHubRulesOK** [reply]
I dont use BitLockerEncryption as a rule (seems unneeded) but what about a file from an USB chip without bit blocker (or will it be included?) is that faster ?
![image](https://github.com/user-attachments/assets/e0a0ce4a-3c83-4e65-912e-3527ace9b1ae)

**Reply by hagamingpc** [reply]
"Included,     but your previous suggestion seemed to work. I uninstalled and then reinstalled, and now it works faster—not very fast, but faster than before."

**Comment by hagamingpc** [comment]
"I am closing the discussion. By the way, thanks for being with me and for your help."
